apps/ledger-live-mobile/src/components/Stake/index.tsx

```diff
@@ -28,21 +27,6 @@ const StakeFlow = ({ route }: Props) => {
 
   const goToAccount = useCallback(
     (account: Account, parentAccount?: Account) => {
-      if (alwaysShowNoFunds) {
```

Comment:
"alwaysShowNoFunds" is the param used to switch between "stake" flow and "no funds" flow.
("Stake" flow has 2 versions: when the user can stake go to "stake", when the user can not stake go to "no funds" ).
If you will visit "stake" flow if user can stake.

(Perhaps I am not seeing something, please, let me know if I am wrong)